test/test_export_vm.py

```diff
@@ -52,7 +55,7 @@ def aws_vm_export_mock():
              3. Returns the mocked ExportImageTask object when calling get_export_image_task()
              4. Returns the mocked Ami object when calling get_ami() (see method get_ami_side_effect() for details
     """
-    aws_access_mock:Union[AwsAccess,MagicMock] = create_autospec(AwsAccess, spec_set=True)
+    aws_access_mock: Union[AwsAccess, MagicMock] = create_autospec(AwsAccess, spec_set=True)
```

Comment:
For union types you also can use the following syntax `AwsAccess | MagicMock` depending on the python version(s)
you use/support you need may need to add this import `from __future__ import annotations`

Reply:
Cool